property-order.js

```diff
@@ -56,7 +77,10 @@ module.exports = [
   "border-top-right-radius",
   "border-bottom-left-radius",
   "border-bottom-right-radius",
+
   "box-shadow",
+  "box-sizing",
```

Comment:
hrmmm i'm not sure where to put this one... makes sense to keep it with `box-shadow` since they share a prefix but i hear your argument.

after reading the docs though, i think this might be the correct location as `box-sizing: border-box` affects the meaning of `width` and `height`, not `margin`.

> `border-box`: The `width` and `height` properties include the padding and border, but not the margin.